validator/rpc/handlers_keymanager.go

```diff
@@ -479,7 +479,7 @@ func (s *Server) ImportRemoteKeys(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 	if isUrlUsed {
-		log.Warnf("Setting web3signer base url for IMPORTED keys is not supported. Prysm only uses the url from --validators-external-signer-url flag for web3signerKeymanagerKind.")
+		log.Warnf("Setting the remote signer base url within the request is not supported. The remote signer url can only be set from the --validators-external-signer-url flag.")
```

Comment:
Is it possible to use `"... %s", flags.XXX.name` instead of this hard coded flag in case we change the flag?